src/main/java/mate/academy/spring/service/RoleService.java

```diff
@@ -0,0 +1,9 @@
+package mate.academy.spring.service;
+
+import mate.academy.spring.model.Role;
+
+public interface RoleService {
+    Role add(Role role);
+
+    Role getByName(Role.RoleName roleName);
```

Comment:
according to the requirements of the task, the input parameter must be of type String
![image](https://github.com/mate-academy/jv-spring-security-part-2/assets/32339836/b1a6d70d-b8fa-45c8-b6bd-462d6981bd6b)